syft/frameworks/torch/hook/hook_args.py

```diff
@@ -131,8 +131,8 @@ def hook_method_args(attr, method_self, args, kwargs):
     return new_self, new_args, kwargs
 
 
-def hook_function_args(attr, args, kwargs, return_args_type=False):
-    """See hook_method_args for details
+def hook_function_with_args(attr, args, kwargs, return_args_type=False):
```

Comment:
Same renaming should apply than:
hook_method_args -> unwrap_args_with_method
hook_function_args -> unwrap_args_with_function

Reply:
Even better: hook_function_args -> unwrap_args_from_function